test/integration/test_job_environments.py

```diff
@@ -27,14 +27,30 @@
 ])
 
 
-class RunsEnvironmentJobs:
+class BaseJobEnvironmentIntegrationTestCase(integration_util.IntegrationTestCase):
+
+    framework_tool_and_types = True
+
+    @classmethod
+    def setUpClass(cls):
+        os.environ["SOME_ENV_VAR"] = "Value from env"
+        super().setUpClass()
+
+    def setUp(self):
+        super().setUp()
+        self.dataset_populator = DatasetPopulator(self.galaxy_interactor)
 
     def _run_and_get_environment_properties(self, tool_id="job_environment_default"):
         with self.dataset_populator.test_history() as history_id:
             self.dataset_populator.run_tool(tool_id, {}, history_id)
             self.dataset_populator.wait_for_history(history_id, assert_ok=True)
             self._check_completed_history(history_id)
-            return self._environment_properties(history_id)
+            job_env_properties = self._environment_properties(history_id)
+            # TODO: Remove the `if` after the Pulsar PR is merged
+            if not isinstance(self, EmbeddedPulsarDefaultJobEnvironmentIntegrationTestCase):
+                # Check that the job hasn't inherited env variables from Galaxy
+                assert job_env_properties.some_env != "Value from env"
```

Comment:
```suggestion
            # Check that the job hasn't inherited env variables from Galaxy
            assert job_env_properties.some_env != "Value from env"
```

Reply:
Thanks, fixed in a commit where I've also updated pulsar-galaxy-lib to the latest version, and rebased.